ec2snap/ec2snap

```diff
@@ -647,6 +656,12 @@ def main():
             print("Don't have permission to read credentials file")
             sys.exit(1)
 
+    if arg.add_tags is not None and len(arg.add_tags) > 0:
+        for elem in arg.add_tags[0]:
+            if elem.find(':') == -1:
+                print("Tag '%s' has wrong format, missing separator ':'" % elem)
```

Comment:
I think the goal was reached. The meaning of the task is completed by allowing adding additional tags to the snapshot.

Replacing the print statements in order to use logger it is a great idea. But im my opinion should be performed in a different task.